### PR TITLE
AM organisation role mapping helm chart config

### PR DESCRIPTION
**Pull request by nitinsahu1**
JIRA link (if applicable)
https://tools.hmcts.net/jira/browse/AM-813

Change description
Onboarding AM organisation role mapping service. This service will spin up the Kubernetes pods at specified times and perform the required actions.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: k8s/aat/common/am/am-org-role-mapping-service.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: am-org-role-mapping-service
+    version: 0.0.1
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
```
Comment (adusumillipraveen) [inline k8s/aat/common/am/am-org-role-mapping-service.yaml:22]: This is not needed if u are using latest java chart

Comment (nitishsingla1234) [reply k8s/aat/common/am/am-org-role-mapping-service.yaml:22]: Hi Praveen, we will remove "useInterpodAntiAffinity: true"

**Comment by nitishsingla1234** [comment]
Hi @adusumillipraveen We have deleted "useInterpodAntiAffinity" property
Can you please have a look.

**Comment by nitishsingla1234** [comment]
Hi @adusumillipraveen We have a couple of new micro services coming up which have a very similar initial setup, including the gradle, database and test cases. So we are using a template derived from a common project structure.
Going forward, we will set it up from Spring Boot Template.

Upgraded the java chart dependency to version 3.0.2.

**Pull request closed without merging**